run-kts.bat

```diff
@@ -0,0 +1,38 @@
+@echo off
+rem Contributor Puzzler runner for Windows
+rem Configure it as an External Tool in IDEA with a keyboard shortcut
+rem Arguments: "$FilePath$" "$FileFQPackage$.$FileNameWithoutExtension$Kt"
+
+setlocal
+
+set FILE=%1
+set CLASS=%2
+
+set FALLBACK_KOTLIN_HOME=%UserProfile%\.IntelliJIdea2018.3\config\plugins\Kotlin\kotlinc\lib
```

Comment:
The version changes over time. It's even different now.

Reply:
Yep, I know; wasn't bothered enough to figure out how to do
https://github.com/angryziber/kotlin-puzzlers/blob/4e42c33b3295e32ea20f7af0dc05a17adec1df6d/run-kts.sh#L9
I was thinking at least have some basic way to actually compile and run them on Windows...

Reply:
Note that one should have KOTLIN_HOME set up